ERCS/erc-7930.md

```diff
@@ -0,0 +1,248 @@
+- While a fresh start on text-encoding schemes could e.g. make use of more characters with base58 or base64 to achieve greater information efficiency and/or abstract which chain the address exists on as an implementation detail, a more familiar approach was chosen since:
+    - It makes the Interoperable Address <-> raw address relationship evident to the user, which will make the cases where the former are not (yet) supported by a dapp or hardware wallet easier to reason with, as users will evidently see the address is the same, but it's the chain&checksum that is being trimmed.
+    - Its extra human-readability will make it more useful as a a stopgap solution until a standard focused on chain and address names is finalized.
+- We chose to support zero-length addresses and chainids to be able to use just one standard to represent both potentially foreign _target addresses_ (e.g. the recipient of a cross-chain message) and also plain addresses or chainids (e.g. to specify the origin-chain hook address of a cross-chain message), for greater uniformity across implementations.
+
+### Comparisons with other standards
+
+#### [CAIP-10]
+[CAIP-10] proposes a standard text format to represent what in this document is defined as target addresses. 
+Since it's a text-only standard, it does not concern itself with the serialization/deserialization of the various chains' address formats, simply using the text representation of addresses already defined by the respective chains. This means it is trivial to add support for chains to [CAIP-10], while doing so with this standard involves defining a serialization scheme.
+
+The above has the drawback of not being able to guarantee canonicity. That is, there could be multiple valid [CAIP-10] addresses pointing to the same target address.
+
+Also, the text format is of little use for smart contracts involved with cross-chain liquidity transfers and message passing due to its reduced data efficiency and lack of canonicity.
+
+Interoperable Names are convertible to [CAIP-10] without even going through the binary representation, so backwards-compatibility with actors expecting [CAIP-10] identifiers should not be an issue.
+
+#### [ERC-7828]
+[ERC-7828] is a text-only representation of addresses with optional resolution into more human-readable strings by use of existing ENS infrastructure for address names, and both the centralized ethereum-lists/chains list and [ERC-7785] for resolution of chain names, while this aims to be a simple but extendable binary-first format.
+
+The interop roadmap is better served by having a standardized binary format for addresses first, which allows the message passing and intents verticals to move forward on a common interface, with a good-enough text representation which is familiar to users and useful for developers, and as a next step develop a chain & address name resolving standard on top of it, leveraging its uniformity and extensibility.
+
+## Test Cases
```

Comment:
I think the examples 3 and 5 don't demonstrate new serialization concepts beyond what 1 and 2 already cover.

- **Example 3** (chain-only): While the spec allows zero-length addresses, this edge case adds complexity without showing practical usage. The parsing mechanics are identical to Example 2, just with `AddressLength = 0`.

- **Example 5** (Arbitrum): This is essentially a variation of Example 1 (EVM chain) with different chain ID encoding length. It doesn't demonstrate new serialization patterns.

I would consider either removing them or repurposing them